action-ssh-to-runner/action.yaml

```
@@ -17,27 +17,70 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: "SSH to runner: Package dependencies for tmate"
+    - name: "SSH to runner: Write tmate config file"
       shell: bash
       run: |
-        ${{ inputs.sudo }} && sudo_cmd="sudo" || sudo_cmd=""
-        # mxschmitt/action-tmate does not have a check to use yum/dnf instead of apt.
-        # For now, all dependencies needed by tmate are already installed.
-        test -f /etc/lsb-release || $sudo_cmd ln -s /bin/true /bin/apt-get
-    - name: "SSH to runner: Print usage information"
+        TMATE_CONF=/tmp/tmate.conf
+        echo "TMATE_CONF=$TMATE_CONF" >> $GITHUB_ENV
+        echo "set -g tmate-server-host ${{ inputs.tmate-server-host }}" >> ${TMATE_CONF}
+        echo "set -g tmate-server-port ${{ inputs.tmate-server-port }}" >> ${TMATE_CONF}
+        echo "set -g tmate-server-rsa-fingerprint ${{ inputs.tmate-server-rsa-fingerprint }}" >> ${TMATE_CONF}
+        echo "set -g tmate-server-ed25519-fingerprint ${{ inputs.tmate-server-ed25519-fingerprint }}" >> ${TMATE_CONF}
+        cat ${TMATE_CONF}
+    - name: "SSH to runner: Install tmate"
       shell: bash
       run: |
-        echo "You can connect to this runner using the ssh command printed in the next step."
-        echo "To connect, you need the ssh key used by your GitHub account."
-        echo "You also need to connected to the VPN."
-        echo "The workflow has been paused."
-        echo "To resume, create a file named 'continue' under $GITHUB_WORKSPACE or under / in the runner."
+        TMATE_VERSION=2.4.0
+        ARCH=i386
```

You sure about the arch? Shouldn't it be `amd64`?
```suggestion
        ARCH=amd64
```